src/components/page/page.scss

```diff
@@ -1,3 +1,12 @@
+html {
+  height: 100%;
+}
+
 .page {
   padding: 0;
+  height: 100%;
+}
+
+.main-container {
```

Comment:
if main container is a block -- why is it in the file 'page.css'? BTW you probably do not need it